### PR TITLE
forward chain example

### PR DESCRIPTION
**Pull request by yantrabuddhi**
forward chaining example from wikipedia implementation

### CODE REVIEW

File: examples/forward_chaining/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+https://en.wikipedia.org/wiki/Forward_chaining
+In this example we have a black box for which we know there is something in it, and it makes croaking sounds and eats flies.
+The objective now is to find the color of the thing in black box.
+we have the following relations defined
+1. If X croaks and X eats flies - Then X is a frog
+2. If X is a frog - Then X is green
+Let's say the thing in black box is named fritz and from above relations we need to deduce its color.
+
+We define the first rule:
+(define rule1
```
Comment (AmeBel) [inline examples/forward_chaining/README.md]: add the github code quoting ``` for clarity, on all the atomese

File: examples/forward_chaining/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	rule1
+	(ConceptNode "wikipedia-fc")
+)
+
+(MemberLink (stv 0.5 1)
+	rule2
+	(ConceptNode "wikipedia-fc")
+)
+Then we run the forward chainer by
+cog-fc source1 wiki
```
Comment (AmeBel) [inline examples/forward_chaining/README.md]: (cog-fc source1 wiki)

Comment (yantrabuddhi) [inline examples/forward_chaining/README.md]: Thanks Amen, I have made the changes.
Regards,
Mandeep

**Comment by AmeBel** [comment]
Thanks

**Comment by williampma** [comment]
It's not really quoting at the right place... just nitpicking :)

**Comment by AmeBel** [comment]
Since, mandeep is on his way to rewrite the atomspace code using atomese, I merged to clear his roadway to it :smiley_cat:

**Comment by ngeiswei** [comment]
I think it should rather be under examples/rule-engine/forwardchainer

**Comment by yantrabuddhi** [comment]
Hi,
 I moved the forward-chaining to rule-engine directory

**Comment by ngeiswei** [comment]
Thanks @yantrabuddhi, ideally you'd mention the new pull request in your message, like "see #177".

**Comment by yantrabuddhi** [comment]
Thanks Nil,
 Next time onward i will do that.
Regards,
Mandeep

**Pull request merged**